ingestion/functions/parsing/taiwan/taiwan.py

```diff
@@ -0,0 +1,122 @@
+import os
+import sys
+from datetime import datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+TAIWAN_LOCATION = {
+    "country": "Taiwan",
+    # Go to wikipedia to debate please, thank you.
+    "geoResolution": "Country",
+    "name": "Taiwan",
+    "geometry": {
+        "longitude": "120.930229378541",
+        "latitude": "23.7779779950014"
+    }
+}
+
+
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+
+    Date filtering API expects mm/dd/YYYYZ format.
+    """
+    date = datetime.strptime(raw_date, "%Y/%m/%d")
+    return date.strftime("%m/%d/%YZ")
+
+
+def convert_gender(raw_gender: str):
+    if raw_gender == "男":
+        return "Male"
+    elif raw_gender == "女":
+        return "Female"
+    else:
+        raise ValueError(f'Unhandled gender: {raw_gender}')
+
+
+def convert_demographics(gender: str, age: str):
+    demo = {}
+    if gender:
+        demo["gender"] = convert_gender(gender)
+    if age:
+        # Brackets are like "35-39" or "over 85".
+        if '-' in age:
+            start, _, end = age.partition("-")
+            demo["ageRange"] = {
+                "start": float(start),
+                "end": float(end),
+            }
+        elif age.isdigit():
+            demo["ageRange"] = {
+                "start": float(age),
+                "end": float(age),
+            }
+        elif age == "70+":
+            demo["ageRange"] = {
+                "start": 70.0,
+            }
+        else:
+            raise ValueError(f'Unhandled age: {age}')
+    return demo or None
+
+
+def convert_notes(immigration_status: str):
+    return (
+        f"Case is an immigrant" if immigration_status == "是"
+        else "Case is from Taiwan")
+
+
+def parse_cases(raw_data_file: str, source_id: str, source_url: str):
+    """Parses G.h-format case data from raw API data.
+    
+    Remarks: No per-case ID is available so we can't dedupe cases.
```

Comment:
Would be nice to add the same comment from the README here too

Reply:
that's the same comment, am I missing something?

Reply:
I meant the comment I requested you to add. But I see it's explained elsewhere, so no need.